docs/fr/README.md

```diff
@@ -0,0 +1,13 @@
+[English](../en)
+
+# Documents sur track-web
+
+Bienvenue dans les documents sur [`track-web`](https://github.com/cds-snc/track-web)!  
+Vous trouverez ici les documents suivants pour vous aider à comprendre le système.
+
+| document | description |
+| -------- | ----------- |
+| [ajouts.md](ajouts.md) | Ce document décrit l’emplacement des principales fonctionnalités pour faciliter les ajouts ou les modifications au système. |
+| [google-analytics.md](google-analytics.md) | Ce document présente les événements personnalisés de Google Analytics que nous avons groupés dans le projet. |
+| [directives-locales.md](directives-locales.md) | Ce document contient des directives étape par étape sur la configuration d’une version de développement du projet à l’échelle locale. |
+| [mise-a-jour-des-trousses.md](mise-à-jour-des-trousses.md) | Ce document est une note de toutes les trousses JavaScript qui ont été incluses dans le dossier statique et doivent être mises à jour lorsque de nouvelles versions sont présentées. |
```

Comment:
Your link `mise-à-jour-des-trousses.md` gives a 404 since the document is actually named `mise-a-jour-des-trousses.md`